src/localization/nav2_amcl/src/amcl_node.cpp

```diff
@@ -642,7 +651,7 @@ boost::recursive_mutex::scoped_lock ml(configuration_mutex_);
   while(!map_received)
   {
     auto result_future = client->async_send_request(req);
-    if (rclcpp::spin_until_future_complete(std::shared_ptr<rclcpp::Node>(this), result_future, std::chrono::seconds(1)) !=
+    if (rclcpp::spin_until_future_complete(std::shared_ptr<rclcpp::Node>(this, [](rclcpp::Node*){}), result_future, std::chrono::seconds(1)) !=
```

Comment:
How does this lambda function with with spin_until_future_complete? Wondering whether it affects CPU utilization.